### PR TITLE
Show icons in Schema Editor, fix #955

### LINKED ISSUE #955: Show icons to discern enums, tables etc.
Should probably use the familiar icons from pgadmin
image

### PR DESCRIPTION
**Pull request by hendi**
This fixes #955 and adds some icons from the pgadmin4 project to the schema editor:

image

I originally added some code to order the DDL statements, e.g. first tables, then indices, then enums etc. But I removed that change since I usually have my DDL statements in an order that makes sense to me (regarding ordering, also have a look at #584). That's also why I changed the code to include comments as well. Thoughs?

### CODE REVIEW

File: IHP/IDE/SchemaDesigner/View/Layout.hs [hunk 1]
```diff
@@ -287,17 +287,67 @@ renderObjectSelector statements activeObjectName = [hsx|
         |]
             where
                 contextMenuId = "context-menu-" <> tshow id
-        renderObject Comment {} id = mempty
```
Comment (mpscholten) [inline IHP/IDE/SchemaDesigner/View/Layout.hs:290]: Why have these empty statements been removed? The idea was that we hide all the statements that the user cannot interact with from the GUI

Comment (hendi) [reply IHP/IDE/SchemaDesigner/View/Layout.hs:290]: My train of thought was: icons look nice -> oh, we only show tables and enums -> let's show more (all) to see more pretty icons

Comment (mpscholten) [reply IHP/IDE/SchemaDesigner/View/Layout.hs:290]: I agree, but in big project's this really wastes a huge amount of screen space (I've tested it)

Comment (hendi) [reply IHP/IDE/SchemaDesigner/View/Layout.hs:290]: We could sort the entries by type (e.g. tables, types, indexes), add a sub-header and by default collapse everything except tables. That's also how pgadmin and dbeaver work.

(FWIW, I'm also fine with simply hiding everything except tables, types and indexes)

Comment (mpscholten) [reply IHP/IDE/SchemaDesigner/View/Layout.hs:290]: This is all too complicated for things that cannot be used from the Schema Designer. Let's keep the behaviour from master and hide everything which cannot be interacted with

**Review by mpscholten — APPROVED** [review]
Thanks :) Good job

**Pull request merged**